indico/modules/events/registration/models/forms.py

```diff
@@ -164,6 +164,11 @@ class RegistrationForm(db.Model):
         nullable=False,
         default=0
     )
+    #: Base registration fee information (used by PayPal)
```

Comment:
let's not mention paypal here and instead also show  it in the normal UI somewhere (tooltip on "registration fee" maybe)